### PR TITLE
Grid Component

### PR DESCRIPTION
**Pull request by bajajcodes**
- Grid layout wrapper is container can 2 items and 3 or mores in column layout fashion.
1. 2 Items in Grid
2. 3 Items in Grid

**FILES TO CHECK** 
> css files only

### CODE REVIEW

**Review by RahullRawat — APPROVED** [review]

File: pages/docs.html [hunk 1]
```diff
@@ -1,118 +1,160 @@
...
+  </head>
+  <body>
     <header class="header">
-        <nav class="navbar">
+      <nav class="navbar">
+        <div class="navbar-brand">
+          <a href="../index.html" class="nav-link">
+            <img
+              src="../assets/logo/logo.svg"
+              alt=""
```
Comment (Sharmaryan) [inline pages/docs.html:20]: alt attribute is empty

File: pages/grid.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="../assets/logo/logo.svg" type="image/svg+xml">
+    <link rel="stylesheet" href="../css/devui.css">
+    <link rel="stylesheet" href="../css/docs.css">
+    <script src="../scripts/navbar.js" defer></script>
+    <script src="../scripts/aside-navbar.js" defer></script>
+    <title>Docs: Alert</title>
+</head>
```
Comment (Sharmaryan) [inline pages/grid.html:13]: it is grid, so titile is suppose to be Docs : Grid

File: pages/navbar.html [hunk 3]
```diff
@@ -1,135 +1,182 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="../assets/logo/logo.svg" type="image/svg+xml" />
+    <link rel="stylesheet" href="../css/devui.css" />
+    <link rel="stylesheet" href="../css/docs.css" />
     <script src="../scripts/navbar.js" defer></script>
     <script src="../scripts/aside-navbar.js" defer></script>
     <title>Docs</title>
-</head>
```
Comment (Sharmaryan) [inline pages/navbar.html:12]: title should be navbar, I think

File: pages/navbar.html [hunk 4]
```diff
@@ -1,135 +1,182 @@
...
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <div class="navbar-brand">
+          <a href="../index.html" class="nav-link">
+            <img
+              src="../assets/logo/logo.svg"
+              alt=""
+              width="40"
```
Comment (Sharmaryan) [inline pages/navbar.html:21]: alt is empty

**Review by Sharmaryan — APPROVED** [review]
left few comments, you can make those changes

File: pages/docs.html [hunk 5]
```diff
@@ -1,118 +1,160 @@
...
+        <h2>Installation</h2>
 
+        <div class="article-component-container">
+          <div class="article-component-header">
+            <h3>Import Link in HTML file</h3>
+            <p>
+              Copy-paste the stylesheet
+              <code class="color-red">&ltlink&gt</code> into your
+              <code class="color-red">&lthead&gt</code> before all other
+              stylesheets to load our CSS.
```
Comment (Mahi24-prog) [inline pages/docs.html:110]: why you are using class as color-red. you can give any relevant name.

File: pages/grid.html [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+<body>
+    <header class="header">
+        <nav class="navbar">
+
+            <div class="navbar-brand">
+                <a href="../index.html" class="nav-link">
+                    <img src="../assets/logo/logo.svg" alt="" width="40" height="40"
+                        class="d-inline-block align-text-top">
+                    Dev Ui
```
Comment (Mahi24-prog) [inline pages/grid.html:23]: give alt attribute to image tag.

**Review by Mahi24-prog — APPROVED** [review]
Go through suggestions once.

File: pages/navbar.html [hunk 7]
```diff
@@ -1,135 +1,182 @@
...
-</body>
-</html>
\ No newline at end of file
+    <footer class="footer">
+      <div class="footer-brand">
+        <div class="navbar-brand footer-logo">
+          <a href="#" class="nav-link">
+            <img
+              src="../../assets/logo/logo.svg"
+              alt=""
```
Comment (pradipgharge) [inline pages/navbar.html:163]: alt attribute is left empty

**Review by pradipgharge — APPROVED** [review]

**Pull request merged**